### PR TITLE
EZP-30138 Solr scoring is not taken into account in the UDW search results list order

### PR DESCRIPTION
**Pull request by kmadejski**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-30138](https://jira.ez.no/browse/EZP-30138)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   |no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

As mentioned in the JIRA ticket, Solr scoring is not taken in the search results list order in UDW. 

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

@sunpietro / @dew326 - guys, could you please review. I don't have sufficient permissions on this repo to ask for the review.

### CODE REVIEW

**Comment by sunpietro** [comment]
@kmadejski please include compiled files as well. Otherwise it won't work on production.

File: src/modules/universal-discovery/services/universal.discovery.service.js [hunk 1]
```diff
@@ -204,7 +204,7 @@ export const findContentBySearchQuery = ({ token, siteaccess }, query, callback)
                 Criteria: {},
                 FacetBuilders: {},
                 SortClauses: {},
-                Filter: { FullTextCriterion: query },
+                Query: { FullTextCriterion: query },
```
Comment (micszo) [inline src/modules/universal-discovery/services/universal.discovery.service.js:207]: That's one slick looking fix! ✨ 
Looking forward to checking if it works. 😉

**Comment by kmadejski** [comment]
@sunpietro done in https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/141/commits/e1a1b67558170e0caf4045818219d37262e078f0.

**Review by tischsoic — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by sunpietro — APPROVED** [review]

**Review by micszo — APPROVED** [review]
Retested successfully with Solr6.
With Solr4 scoring doesn't work in UDW (it does in Content / Search).
Also patch from PR loads only on 2.2.3, on 2.3.2 and 2.4.1 had to use branch.

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by kmadejski** [comment]
@lserwatka With pleasure, but unfortunately I have not enough permission on this repository to do so.

**Comment by lserwatka** [comment]
@kmadejski Fixed!

**Comment by kmadejski** [comment]
@lserwatka Thanks! Done in: https://github.com/ezsystems/ezplatform-admin-ui-modules/commit/e48e62b827cbdc0285dca104ee4261d1aa552363 / https://github.com/ezsystems/ezplatform-admin-ui-modules/commit/9420fee6303eca0f51999d22a980e718c7583f21 / https://github.com/ezsystems/ezplatform-admin-ui-modules/commit/968a311b18c02c612770cbb05a6367783c94c653.

**Pull request merged**